packages/orbit-components/src/LinkList/README.md

```diff
@@ -40,14 +40,25 @@ Table below contains all types of the props available in LinkList component.
 
 ### spacing
 
-| name         | size on `992px - ∞` |
-| :----------- | :------------------ |
-| `"none"`     | `null`              |
-| `"XXXSmall"` | `2px`               |
-| `"XXSmall"`  | `4px`               |
-| `"XSmall"`   | `8px`               |
-| `"small"`    | `12px`              |
-| `"medium"`   | `16px`              |
-| `"large"`    | `24px`              |
-| `"XLarge"`   | `32px`              |
-| `"XXLarge"`  | `40px`              |
+| name                                        | size on `992px - ∞` |
+| :------------------------------------------ | :------------------ |
+| `"none"`                                    | `null`              |
+| `"XXXSmall"` - **deprecated (use `"50"`)**  | `2px`               |
+| `"XXSmall"` - **deprecated (use `"100"`)**  | `4px`               |
+| `"XSmall"` - **deprecated (use `"200"`)**   | `8px`               |
+| `"small"` - **deprecated (use `"300"`)**    | `12px`              |
+| `"medium"` - **deprecated (use `"400"`)**   | `16px`              |
+| `"large"` - **deprecated (use `"600"`)**    | `24px`              |
+| `"XLarge"` - **deprecated (use `"800"`)**   | `32px`              |
+| `"XXLarge"` - **deprecated (use `"1000"`)** | `40px`              |
+| `"50"`                                      | `2px`               |
+| `"100"`                                     | `4px`               |
```

Comment:
value for `space150` is missing here.

Reply:
Well spotted. It was missing in all of them, actually. Added. Thanks